app/presenters/facet_link_group_presenter.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+##
+# Display logic for facet link groups(facet entry points).
+#
+# Responsible for generating data hashes for the Mustache templates from
+# [Europeana's styleguide](https://github.com/europeana/europeana-styleguide-ruby)
+# to display facet entry points on browse based landing pages.
+class FacetLinkGroupPresenter
+  include Facet::Labelling
+  include FacetsHelper
+  include UrlHelper
+  delegate :t, to: I18n
```

Comment:
it does. theoretically we could leave out the delegation since we pass missing methods to the controller anyway.